framework/filters/auth/CompositeAuth.php

```diff
@@ -85,6 +86,13 @@ public function authenticate($user, $request, $response)
                 continue;
             }
 
+            $authUser = $auth->user;
```

Comment:
@samdark can this PR be reverted?
There is no `user` property on SessionAuth. This breaks implementations properly implementing the interface but not extending from AuthMethod.

Reply:
@SamMousa means this SessionAuth: https://github.com/collecthor/yii2-session-auth